README.md

```diff
@@ -8,141 +8,337 @@
+                    // Configure Gradle project's repositories
+                    jf 'gradle-config --repo-resolve libs-release --repo-deploy libs-release-local'
 
-Please make sure to submit pull requests to *https://github.com/jfrog/jenkins-jfrog-plugin* only.
+                    // Install and publish project
+                    jf 'gradle clean artifactoryPublish'
+                }
+            }
+        }
 
-## Tests
-### Unit tests
-To run unit tests execute the following command:
-```
-> mvn clean test
+        stage('Publish build info') {
+            steps {
+                jf 'rt build-publish'
+            }
+        }
+    }
+}
 ```
 
-### Integration tests
-#### Running integration tests
-Before running the integration tests, set the following environment variables.
+</details>
+<details>
+  <summary>npm Example</summary>
+
+```groovy
+pipeline {
+    agent any
+    tools {
+        jfrog 'cli'
+    }
+    stages {
+        stage('Clone') {
+            steps {
+                git branch: 'master', url: "https://github.com/jfrog/project-examples.git"
+            }
+        }
+
+        stage('Exec npm commands') {
+            steps {
+                dir('npm-example') {
+                    // Configure npm project's repositories
+                    jf 'npm-config --repo-resolve npm-remote --repo-deploy npm-local'
 
-*JFROG_URL*<br>
-*JFROG_USERNAME*<br>
-*JFROG_PASSWORD*<br>
-*JFROG_ADMIN_TOKEN*<br>
+                    // Install dependencies
+                    jf 'npm install'
 
-Run the integration tests.
+                    // Pack and deploy the npm package
+                    jf 'npm publish'
+                }
+            }
+        }
+
+        stage('Publish build info') {
+            steps {
+                jf 'rt build-publish'
+            }
+        }
+    }
+}
 ```
-> mvn clean verify -DskipITs=false
+
+</details>
+
+<details>
+  <summary>Go Example</summary>
+
+```groovy
+pipeline {
+    agent any
+    tools {
+        jfrog 'cli'
+    }
+    stages {
+        stage('Clone') {
+            steps {
+                git branch: 'master', url: "https://github.com/jfrog/project-examples.git"
+            }
+        }
+
+        stage('Exec Go commands') {
+            steps {
+                dir('golang-example/hello/') {
+                    // Configure Go project's repositories
+                    jf 'go-config --repo-resolve go-remote --repo-deploy go-local'
+
+                    // Build the project with go and resolve the project dependencies from Artifactory
+                    jf 'go build'
+
+                    // Publish version v1.0.0 of the package to the go-local repository in Artifactory
+                    jf 'go-publish v1.0.0'
+                }
+            }
+        }
+
+        stage('Publish build info') {
+            steps {
+                jf 'rt build-publish'
+            }
+        }
+    }
+}
 ```
+
+</details>
+
```

Comment:
Since the examples we added do not cover all the capabilities for JFrog CLI, I suggest adding the following paragraph here - 

These examples demonstrate only a fraction of the capabilities of JFrog CLI.  Please refer to the [JFrog CLI documentation](https://www.jfrog.com/confluence/display/CLI/JFrog+CLI) for additional information.